code-challenges.js

```diff
@@ -13,21 +26,53 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+let numsOnly = (value) => {
+    if(typeof(value)=== 'number' && value % 2 !== 0){
+        return value
+    }
+}
```

Comment:
Awesome! I think I would want to point out that when you're creating functions like this, it is important that you are using CONST rather than LET. 

Const keeps our function from being mutated later on and also allows it to be in the global scope, while LET allows the function to be mutated later on and could cause a lot of problems down the road.